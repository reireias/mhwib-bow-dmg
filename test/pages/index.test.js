import { shallowMount, createLocalVue } from '@vue/test-utils'
import Vue from 'vue'
import Vuex from 'vuex'
import Vuetify from 'vuetify'
import target from '@/pages/index'
import { createStore } from '@/test/helper'

Vue.use(Vuetify)
const localVue = createLocalVue()
localVue.use(Vuex)
localVue.use(Vuetify)

describe('index', () => {
  let store
  const actions = {
    loadLocal: jest.fn()
  }
  beforeEach(() => {
    store = createStore({}, actions)
  })

  it('should be vue instance', () => {
    const vuetify = new Vuetify()
    const wrapper = shallowMount(target, {
      localVue,
      store,
      vuetify
    })
    expect(wrapper.isVueInstance()).toBeTruthy()
    // TODO
    // expect(wrapper.html()).toMatchSnapshot()
  })
})
