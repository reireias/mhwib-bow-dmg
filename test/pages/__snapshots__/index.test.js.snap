// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should be vue instance 1`] = `
"<v-tabs-stub activeclass=\\"\\" fixedtabs=\\"true\\" nexticon=\\"$next\\" previcon=\\"$prev\\" slidersize=\\"2\\">
  <v-tab-stub ripple=\\"true\\">計算</v-tab-stub>
  <v-tab-stub ripple=\\"true\\">マイセット</v-tab-stub>
  <v-tab-item-stub>
    <v-container-stub tag=\\"div\\">
      <v-row-stub tag=\\"div\\" justify=\\"center\\">
        <div class=\\"title secondary--text\\">MHW:Iceborne</div>
      </v-row-stub>
      <v-row-stub tag=\\"div\\" justify=\\"center\\">
        <div class=\\"display-1 secondary--text\\">弓専用ダメージ計算機</div>
      </v-row-stub>
      <v-row-stub tag=\\"div\\">
        <weapon-stub></weapon-stub>
      </v-row-stub>
      <v-divider-stub></v-divider-stub>
      <v-row-stub tag=\\"div\\">
        <buff-stub></buff-stub>
      </v-row-stub>
      <v-divider-stub></v-divider-stub>
      <v-row-stub tag=\\"div\\">
        <skill-stub></skill-stub>
      </v-row-stub>
      <v-divider-stub></v-divider-stub>
      <v-row-stub tag=\\"div\\">
        <motion-stub></motion-stub>
      </v-row-stub>
      <v-divider-stub></v-divider-stub>
      <v-row-stub tag=\\"div\\">
        <target-stub></target-stub>
      </v-row-stub>
      <v-divider-stub></v-divider-stub>
      <v-row-stub tag=\\"div\\">
        <result-stub></result-stub>
      </v-row-stub>
    </v-container-stub>
  </v-tab-item-stub>
  <v-tab-item-stub>
    <my-set-stub></my-set-stub>
  </v-tab-item-stub>
</v-tabs-stub>"
`;
